beast/physicsmodel/dust/extinction.py

```diff
@@ -567,3 +571,87 @@ def get_Rv(self, Rv_A, f_A=0.5):
 
         Rv_B = self.BLaw.Rv
         return 1. / (f_A / Rv_A + (1 - f_A) / Rv_B)
+
+
+class Generalized_RvFALaw(Gordon16_RvFALaw):
```

Comment:
I think this should inherit ExtinctionLaw, not Gordon16_RvFALaw.  Unless you are using something from the Gordon16_RvFALaw.

Reply:
I'm inheriting Gordon16_RvFALaw in order to not duplicate the get_Rv and get_Rv_A functions.